Core/GDCore/Extensions/Builtin/SpriteExtension/SpriteObject.cpp

```diff
@@ -25,8 +25,7 @@ namespace gd {
 
 Animation SpriteObject::badAnimation;
 
-SpriteObject::SpriteObject(gd::String name_)
-    : Object(name_), updateIfNotVisible(false) {}
+SpriteObject::SpriteObject() : updateIfNotVisible(false) {}
```

Comment:
I kept the classes names, but they should probably be renamed. Maybe in another PR?

Reply:
Probably in another PR, after we've proven this PR does not introduce any regression :)